cultcargo/images/wsclean/Dockerfile.build

```diff
@@ -1,17 +1,17 @@
-FROM {REGISTRY}/base-cult:base-{BUNDLE_VERSION}
+FROM {REGISTRY}/base-cult:kern-{kern_version}-{BUNDLE_VERSION}
 
-RUN apt-get update && \
-    apt-get -y install casacore-dev libgsl-dev libhdf5-dev \
+RUN apt-get update && apt-get -y upgrade && \
+    apt-get -y install cmake casacore-dev libgsl-dev libhdf5-dev \
         libfftw3-dev libboost-dev \
         libboost-date-time-dev libboost-filesystem-dev \
         libboost-program-options-dev libboost-system-dev \
         libcfitsio-dev cmake g++ cmake \
-        libblas-dev liblapack-dev && \
-    rm -rf /var/lib/apt/lists/* \
+        libblas-dev liblapack-dev \
+    && rm -rf /var/lib/apt/lists/* \
     && rm -rf /var/cache/apt/archives 
 
 RUN git clone https://gitlab.com/aroffringa/wsclean.git && \
-    (cd wsclean && git checkout {tag}) && \
+    (cd wsclean && git checkout v{VERSION}) && \
     mkdir build && \
     (cd build && cmake ../wsclean && make -j16) && \
```

Comment:
Should we not set `-DTARGET_CPU` to something safe (e.g. `hasswell`) here to make the image more portable?